packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -71,9 +76,76 @@ export type CollapsibleProps = {
 } & EnsembleWidgetProps &
   HasItemTemplate;
 
+const Panel = ({
+  scope,
+  setTemplateItems,
+  template,
+}: {
+  scope: CustomScope;
+  template: CollapsibleItem;
+  setTemplateItems: React.Dispatch<React.SetStateAction<CollapsibleItem[]>>;
+}): null => {
+  const evaluated = useEvaluate({ ...template }, { context: scope });
+  const itemAddedRef = useRef(0);
+
+  const evaluatedItem = useMemo(() => {
+    const { key, label, children } = evaluated as {
+      [key: string]: Expression<string>;
```

Comment:
This cast shouldn't be needed. The object type is inferred based on what you pass in.

Reply:
👍